spec/models/series_spec.rb

```diff
@@ -110,4 +152,46 @@
       end
     end
   end
+
+  describe "#tags_general" do
+    it "includes tags on unrestricted works" do
+      series.works = [unrestricted_work]
+      series.reload
+      expect(series.tags_general).to include(*unrestricted_work.tags.pluck(:name))
+    end
+
+    it "includes tags on restricted works" do
+      series.works = [restricted_work]
+      series.reload
+      expect(series.tags_general).to include(*unrestricted_work.tags.pluck(:name))
```

Comment:
```suggestion
      expect(series.tags_general).to include(*restricted_work.tags.pluck(:name))
```

Bad sign though. Could you give the three works unique (canonical for filters_*?) tags like you did in #5184?